azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -93,7 +93,7 @@ private void sendSlaAlertEmail(SlaOption slaOption, String slaMessage) {
           super.createEmailMessage(subject, "text/html", emailList);
 
       message.setBody(body);
-
+      message.setTLS(tls);
```

Comment:
I would say this is not the best fix because it still leaves other places like `SendEmailAction` broken.

See this alternative fix for the same issue: https://github.com/azkaban/azkaban/pull/673/files